libs/wingsdk/.projenrc.ts

```diff
@@ -189,18 +189,42 @@ function disallowImportsRule(target: Zone, from: Zone): DisallowImportsRule {
   };
 }
 
-// Prevent unsafe imports between preflight and inflight and simulator code
 project.eslint!.addRules({
+  // Prevent unsafe imports between preflight and inflight and simulator code
   "import/no-restricted-paths": [
     "error",
     {
       zones: [
-        // avoid importing inflight or simulator code into preflight code since
+        // Avoid importing inflight or simulator code into preflight code since
         // preflight code gets compiled with JSII
         disallowImportsRule(Zone.PREFLIGHT, Zone.INFLIGHT),
       ],
     },
   ],
+  // Makes sure that all methods and properties are marked with the right member accessibility- public, protected or private
+  "@typescript-eslint/explicit-member-accessibility": [
+    "error",
+    {
+      accessibility: "explicit",
+      overrides: {
+        accessors: "off",
+        constructors: "off",
+        methods: "explicit",
+        properties: "explicit",
+        parameterProperties: "explicit",
+      },
+    },
+  ],
+  // Makes sure comments and doc strings are capitalized
+  "capitalized-comments": [
+    "error",
+    "always",
+    {
+      ignoreConsecutiveComments: true,
+      ignorePattern: "pragma|ignored",
+      ignoreInlineComments: true,
+    },
+  ],
```

Comment:
Can we change this to only apply to block style comments (`/* ... */`)? The other comments (`//`) aren't generated in the public APIs like our docs or the Wing IDE extension, so I don't think the capitalization matters as much

Reply:
yes, I fixed it and created another PR (to prevent rolling back the changes to the line comments) 
here it is: https://github.com/winglang/wing/pull/3725